server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/pom.xml

```diff
@@ -65,7 +65,7 @@
         <dependency>
             <groupId>com.softwaremill.sttp.client3</groupId>
             <artifactId>okhttp-backend_${scala.base}</artifactId>
-            <version>3.0.0</version>
+            <version>3.3.18</version>
```

Comment:
Suggestion commit name: `xmlunit 2.6.3 -> 2.8.4 & okhttp 3.0.0 -> 3.3.18`

Reply:
okhttp is just a driver here, the core engine is sttp.